transports/quic/src/config.rs

```diff
@@ -83,8 +86,17 @@ impl Config {
             // Ensure that one stream is not consuming the whole connection.
             max_stream_data: 10_000_000,
             keypair: keypair.clone(),
+            path_mtu_discovery_config: Some(Default::default()),
         }
     }
+
+    pub fn path_mtu_discovery_config(
+        mut self,
+        path_mtu_discovery_config: Option<MtuDiscoveryConfig>,
```

Comment:
This adds `quinn` to the public API which means every dependency bump will be a breaking change.

If all we need for now is disabling path MTU discovery, can we add that as an option instead, i.e. `disable_path_mtu_discovery`.

Reply:
The type can be re-exported, but it is still a potential source of breaking changes. Changed public method to `disable_path_mtu_discovery` as suggested.